nio.json

```diff
@@ -0,0 +1,182 @@
+{
+  "non_editable": {             // editable @ admin/owner level?
+    "persistence": {
```

Comment:
Didn't we say persistence would not be in here...

Reply:
Correct, persistence variables would be based on environment.